src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -254,19 +285,84 @@ public Config(IGerritHudsonTriggerConfig config) {
         enableProjectAutoCompletion = config.isEnableProjectAutoCompletion();
         projectListFetchDelay = config.getProjectListFetchDelay();
         projectListRefreshInterval = config.getProjectListRefreshInterval();
-        if (config.getCategories() != null) {
-            categories = new LinkedList<VerdictCategory>();
-            for (VerdictCategory cat : config.getCategories()) {
-                categories.add(new VerdictCategory(cat.getVerdictValue(), cat.getVerdictDescription()));
-            }
-        }
+
+
+        assertDefaultCategories();
+
         if (config.getReplicationConfig() != null) {
             replicationConfig = new ReplicationConfig(config.getReplicationConfig());
         }
         watchdogTimeoutMinutes = config.getWatchdogTimeoutMinutes();
         watchTimeExceptionData = addWatchTimeExceptionData(config.getExceptionData());
     }
 
+    public static Integer getIntegerFromString(String str) {
+        if (NumberUtils.isNumber(str))
```

Comment:
`NumberUtils.isNumber("1.123")` is `true` so can potentially still throw a `NumberFormatException`.

```suggestion
        try {
            return Integer.parseInt(str);
        } catch (NumberFormatException nfe) {
            return null;
        }
```

Reply:
Done